### ISSUE TITLE
Personen die bereits in einer Lerngruppe sind oder bereits einen Request offen haben, sollten nicht erneut einen join request stellen können.

### ISSUE THREAD

**Issue by ffaadd** [FU-Hagen-Discord/fernuni-bot#180]
s.o.

**Comment by Lou-M** [comment]
War die Anfrage nicht auch, das es pro Person nur eine aktive Beitrittsanfrage gleichzeitig geben darf?

**Comment by dnns01** [comment]
Ja, beides klingt gut.

**Comment by ffaadd** [comment]
Nein. *Diese* Anfrage war, das jemand bereits in der Lerngruppe ist und nochmal `!lg join` eingegeben hat ;-p. Das hab ich per DM bekommen. Eine andere Anfrage war im QA System von dnns. Aber ich glaube beide Anfragen sind so nah beieinander, dass ein Ticket dafür reicht.
